splash/browser_tab.py

```diff
@@ -827,11 +839,21 @@ def get(self, url, callback, headers=None, follow_redirects=True):
 
     def _send_request(self, url, callback, method='GET', body=None,
                       headers=None):
-        # XXX: The caller must ensure self._delete_reply is called in a callback.
-        if method != 'GET':
-            raise NotImplementedError()
-        request = self.request_obj(url, headers=headers)
-        reply = self.network_manager.get(request)
+        # XXX: The caller must ensure self._delete_reply is called in a callback
+        request = self.request_obj(url, headers=headers, body=body)
+
+        operation = getattr(self.network_manager, method.lower(), None)
```

Comment:
I think we should be explicit here, i.e. no use getattr. `method` can be validated somewhere earlier, but if not we'll allow to call any network manager method here.